frontend/src/components/dashboard/ProgressList.tsx

```diff
@@ -181,17 +181,22 @@ const ProgressList = ({
     isLoading: isLoadingSolutions,
   } = useAllSessionSolutions(classId, sessionId);
 
-  const progress = useMemo(() => {
-    if (!klass || !session || !solutions) {
-      return [];
-    }
+  const studentList = useMemo(() => {
```

Comment:
`studentIds` is more accurate I would say. You can rename the variable within the hook to sth like `studentIdSet`